mainwindow.cpp

```diff
@@ -936,7 +937,7 @@ void MainWindow::fillSettingsWithDefaults()
     QStringList possiblePaths;
     QString john;
     // Find john on system path, which is determined by PATH variable
-    QString johnSystemPath = QStandardPaths::findExecutable("john", QStringList());
+    /*QString johnSystemPath = QStandardPaths::findExecutable("john", QStringList());
```

Comment:
We can no longer use QStandardPaths, kind of sad ... The oldest equivalent seems to be QDesktopServices but I didn't find findExecutable stuff yet.